endpoints/_includes/functions.php

```diff
@@ -764,7 +762,11 @@ function getAdminBMLTRootServer()
 
 function getBMLTRootServer()
 {
-    return setting('bmlt_root_server');
+    if(setting('tomato_helpline_routing')) {
```

Comment:
This would affect non-helpline requests.  Not comfortable with this change.

Reply:
The only references to this function now is within meeting searches - all other searches refer to getAdminBMLTRootServer

Reply:
We should not conflate a setting called `tomato_helpline_routing` with meeting searches.  If someone wants to use tomato for their meeting searches they should be deliberate in that by setting the server to that root server.

Reply:
Originally this was its own setting but you recommended consolidating the tomato related settings 

I don’t think requiring people to know the tomato server url should be necessary for enabling tomato meeting search - a setting keeps it simple. Easy enough to break back out to its own tomato_meeting_search setting

Reply:
The change you are proposing has nothing to do with helpline routing.  I'd like to keep this pull request down to material changes to the topic of discussion which is tomato helpline routing.